pkg/cmd/openshift-controller-manager/controller/interfaces.go

```diff
@@ -78,7 +78,7 @@ func NewControllerContext(
 	if err != nil {
 		return nil, err
 	}
-	configClient, err := configclient.NewForConfig(clientConfig)
+	configClient, err := configclient.NewForConfig(nonProtobufConfig(clientConfig))
```

Comment:
@bparees this fixes a bug that prevented the controller from listing and watching the config objects.